pillar.example

```diff
@@ -277,7 +283,45 @@ nginx:
   # Passenger configuration
   # Default passenger configuration is provided, and will be deployed in
   # /etc/nginx/conf.d/passenger.conf
+  # Passenger conf can be retrieved by TOFS ( Fallback to nginx.conf )
   passenger:
     passenger_root: /usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini
     passenger_ruby: /usr/bin/ruby
     passenger_instance_registry_dir: /var/run/passenger-instreg
+
+  tofs:
+    # The files_switch key serves as a selector for alternative
+    # directories under the formula files directory. See TOFS pattern
+    # doc for more info.
+    # Note: Any value not evaluated by `config.get` will be used literally.
+    # This can be used to set custom paths, as many levels deep as required.
+    files_switch:
+      - any/path/can/be/used/here
+      - id
+      - role
+      - osfinger
+      - os
+      - os_family
+    # All aspects of path/file resolution are customisable using the options below.
+    # This is unnecessary in most cases; there are sensible defaults.
+    # path_prefix: template_alt
+    # dirs:
+    #   files: files_alt
+    #   default: default_alt
+    # source_files:
+    #   template-config-file-file-managed:
+    #     - 'example_alt.tmpl'
+    #     - 'example_alt.tmpl.jinja'
+    source_files:
+      nginx_config_file_managed:
+        - 'another_file_name.conf'
+        - 'another_tpl_name.jinja'
+      passenger_config_file_managed:
+        - 'another_file_name.conf'
+        - 'another_tpl_name.jinja'
+      nginx_snippet_file_managed:
+        - 'another_file_name.conf'
+        - 'another_tpl_name.jinja'
+      server_conf_file_managed:
+        - 'another_file_name.conf'
+        - 'another_tpl_name.jinja'
```

Comment:
Could we use this for this whole section instead?

```sls
  tofs:
    # The files_switch key serves as a selector for alternative
    # directories under the formula files directory. See TOFS pattern
    # doc for more info.
    # Note: Any value not evaluated by `config.get` will be used literally.
    # This can be used to set custom paths, as many levels deep as required.
    # files_switch:
    #   - any/path/can/be/used/here
    #   - id
    #   - role
    #   - osfinger
    #   - os
    #   - os_family
    # All aspects of path/file resolution are customisable using the options below.
    # This is unnecessary in most cases; there are sensible defaults.
    # path_prefix: 'alt_nginx'
    # dirs:
    #   files:     'alt_files'
    #   default:   'alt_default'
    source_files:
      nginx_config_file_managed:
        - alt_nginx.conf
      passenger_config_file_managed:
        - alt_nginx.conf
      server_conf_file_managed:
        - alt_server.conf
      nginx_systemd_service_file:
        - alt_nginx.service
      nginx_snippet_file_managed:
        - alt_server.conf
```